src/internet_identity/src/active_anchor_stats.rs

```diff
@@ -62,22 +76,100 @@ fn update_active_anchor_counter(
     }
 }
 
-fn new_active_anchor_statistics(time: Timestamp) -> ActiveAnchorStatistics {
+#[derive(Eq, PartialEq)]
+pub enum IIDomain {
+    Ic0AppDomain,
+    InternetComputerOrgDomain,
+}
+
+impl IIDomain {
+    pub fn is_same_domain(&self, activity: &DomainActivity) -> bool {
+        match self {
+            IIDomain::Ic0AppDomain => matches!(activity, DomainActivity::Ic0App),
+            IIDomain::InternetComputerOrgDomain => {
+                matches!(activity, DomainActivity::InternetComputerOrg)
+            }
+        }
+    }
+
+    pub fn other_domain(&self) -> IIDomain {
+        match self {
+            IIDomain::Ic0AppDomain => IIDomain::InternetComputerOrgDomain,
+            IIDomain::InternetComputerOrgDomain => IIDomain::Ic0AppDomain,
+        }
+    }
+}
+
+impl TryFrom<&str> for IIDomain {
+    type Error = ();
+
+    fn try_from(origin: &str) -> Result<Self, Self::Error> {
+        match origin {
+            IC0_APP_ORIGIN => Ok(IIDomain::Ic0AppDomain),
+            INTERNETCOMPUTER_ORG_ORIGIN => Ok(IIDomain::InternetComputerOrgDomain),
+            _ => Err(()),
+        }
+    }
+}
+
+/// Increases the counter on a window for a single II domain if the there was no activity before.
+///
+/// If there has been activity on an II domain and this is activity on the _other_ domain, then
```

Comment:
"the _other_ domain" here can be (mis)understood as referring to `DomainActivity::Other`, while it refers indeed to the other **II** domain, right?  Maybe rephrase the comment and/or rename `DomainActivity::Other`? (as the name also "collides" with `IIDomain::other_domain()`

Reply:
I did all of those things, thanks for pointing it out:
* renamed `DomainActivity::Other` to `DomainActivity::NonIIDomain`
* renamed `IIDomain::other_domain` to `IIDomain::other_ii_domain`
* clarified the comment to say "other II domain"